src/views/FieldReportDetails/index.tsx

```diff
@@ -0,0 +1,391 @@
+                                                            className={styles.icon}
+                                                        />
+                                                        <div className={styles.label}>
+                                                            {action.name}
+                                                        </div>
+                                                    </div>
+                                                ),
+                                            )}
+                                        </div>
+                                    ),
+                                )}
+                                {actionTaken.summary}
+                            </Container>
+                        ),
+                    )}
+                    <Container
+                        heading={strings.contactTitle}
+                        withHeaderBorder
+                        childrenContainerClassName={styles.contactList}
+                    >
+                        {fieldReportResponse?.contacts?.map((contact) => (
+                            <div
+                                key={contact.id}
+                                className={styles.contact}
+                            >
+                                <div className={styles.type}>
+                                    {contact.ctype}
+                                </div>
+                                <div className={styles.information}>
+                                    {joinList([
+                                        isTruthyString(contact.name) ? contact.name : undefined,
+                                        isTruthyString(contact.title) ? contact.title : undefined,
+                                        isTruthyString(contact.email) ? (
+                                            <Link
+                                                to={`mailto:${contact.email}`}
+                                            >
+                                                {contact.email}
+                                            </Link>
+                                        ) : undefined,
+                                    ].filter(isDefined), ', ')}
+                                </div>
+                            </div>
+                        ))}
+                    </Container>
+                    <Container
+                        heading={strings.externalPartnersSupportedActivitiesLabel}
+                        withHeaderBorder
+                    >
+                        {fieldReportResponse?.external_partners.map(
+                            (partner) => (
+                                <div key={partner.id}>
+                                    {partner?.name}
+                                </div>
+                            ),
+                        )}
+                    </Container>
```

Comment:
Should we handle cases where external_partners is empty @frozenhelium ?

Either show an empty message or not show the container at all?

Reply:
Should hide the container